### ISSUE TITLE
HLS playback on iO14 is too heavy!

### ISSUE THREAD

**Issue by newforestar** [mozilla/hubs#3069]
I've discovered the HLS playback issue in iOS14.
I've tried many m3u8 but they all failed to play properly.
The problem isn't that it can't be played back, it's that it's very heavy and the video is choppy.

I conducted an experiment to see what the cause was:

iOS safari plays HLS natively with the video tag.
So, I pasted the video tag directly into ui-root.js.

As a result, the fps dropped drastically below 30 fps on iOS14, even though I was just playing the video tag.
Of course, the video is interrupted and won't play back properly.
On iOS13, I was able to play back a video in the same situation successfully at 60fps.

In conclusion, playing hls with the video tag in the hubs  will result in a significant drop in frame rate.

I'd like to know if it's an issue with iOSsafari or an issue with hubs.
And I would like to know if there is a way around this problem.

**Comment by takahirox** [comment]
Hi, thanks for the problem report.

Recently iOS 15 has been released. Do you mind if the problem is still reproducible on iOS 15 if you have iOS 15 devices or you can upgrade your iOS?
